src/components/MainView.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState} from "react";
+import CityWeatherCard from "./CityWeatherCard";
+import styles from "../styles/MainView.module.css";
```

Comment:
Generally speaking for 90% of codebases the styles would sit at the same level as the component.
E.G:
```
 - src
    - views
       - MainView
          - index.ts // exports MainView
          - MainView.tsx // component
          - MainView.module.css // styles
```